public/js/pimcore/asset/tree.js

```diff
@@ -363,7 +363,11 @@
  
          if(dataTransfer["items"] && dataTransfer.items[0] && dataTransfer.items[0].webkitGetAsEntry) {
              // chrome
-             var traverseFileTree = function (item, path) {
+             var traverseFileTree = function (item, path, depth = 0) {
+                 if (depth > 100) {
```

Comment:
Not sure what value could make more sense or if it would crash as well before that, maybe
```suggestion
                 if (depth > 50) {
```